.github/workflows/style.yml

```diff
@@ -19,16 +16,21 @@ jobs:
     name: Style Checker
     runs-on: ubuntu-latest
     steps:
+      # Unfortunately there doesn't seem to be a way to
+      # do this without an extra step.
+      - name: Get depth
+        run: |
+          echo "DEPTH=$(( ${{ github.event.pull_request.commits }} + 1 ))" >> $GITHUB_ENV
```

Comment:
I'm surprised we need this.

Reply:
Same here.
I would've thought there would at least be basic mathematical operators here: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#operators